packages/config/config/devices/0x0371/zw187.json

```
@@ -6,96 +6,66 @@
 	"devices": [
 		{
 			"productType": "0x0002",
-			"productId": "0x00bb"
+			"productId": "0x00bb",
+			"zwaveAllianceId": [3620, 3694, 3736]
 		},
 		{
 			"productType": "0x0102",
-			"productId": "0x00bb"
+			"productId": "0x00bb",
+			"zwaveAllianceId": 3785
+		},
+		{
+			"productType": "0x0202",
+			"productId": "0x00bb",
+			"zwaveAllianceId": 3786
 		}
 	],
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
 	},
-	"supportsZWavePlus": true,
 	"associations": {
 		"1": {
 			"label": "Lifeline",
-			"description": "Lifeline group. (1) Device Reset Locally Notification: Issued when Factory Reset is performed. (2) Battery Report: lssued periodically to report the current battery level; lssued when battery becomes low. (3) Sensor Binary Report: Issued when Magnet is away or near. (4) Notification Report (Type=0x06; Event=0x16): Issued when sensor state is changed to be Open Status. (5) Notification Report (Type=0x06; Event=0x17): Issued when sensor state is changed to be Close Status. (6) Notification Report (Type=0x08; Event-0x01): Issued when battery voltage change exceeds 20%. (7) Notification Report (Type-0x09; Event=0x04; Param=0x55): Issued when reset due to Watch-Dog Timeout.",
 			"maxNodes": 5,
 			"isLifeline": true
 		},
 		"2": {
-			"label": "On/Off control (Access)",
-			"description": "Issue Basic Set when magnet is away or near. (The Basic Set Value is determined by Confiquration Parameter 3)",
+			"label": "Group 2",
 			"maxNodes": 5
 		}
 	},
 	"paramInformation": {
 		"1": {
-			"label": "Binary Sensor Report",
-			"description": "Enable/Disable Binary Sensor Report. Allow for backward compatibility to report Binary if Notification Report cannot be used for status changes. When enabled the Sensor Binary Report will NOT be issued via Lifeline when the magnet is away or near. When disabled the Sensor Binary Report will be issued via Lifeline when the magnet is away or near.",
-			"unsigned": true,
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 1,
-			"defaultValue": 0,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Disable",
-					"value": 0
-				},
-				{
-					"label": "Enable",
-					"value": 1
-				}
-			]
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Binary Sensor Report"
 		},
 		"2": {
-			"label": "Reverse Sensor Reports",
-			"description": "Reverse both Sensor Binary Report and Notification Report.",
-			"unsigned": true,
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 1,
-			"defaultValue": 0,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Open when magnet is away, close when magnet is near.",
-					"value": 0
-				},
-				{
-					"label": "Closed when magnet is away, open when magnet is near.",
-					"value": 1
-				}
-			]
+			"$import": "../0x0086/templates/aeotec_template.json#invert_state_report"
 		},
 		"3": {
 			"label": "Association Group 2 Settings",
```

I think repeating "Basic Set" in each option is a bit redundant. How about this?
```suggestion
			"label": "Association Group 2: Basic Set Value",
```
and then shorten the options.